share/models/ingest.py

```diff
@@ -109,6 +109,9 @@ class SourceConfig(models.Model):
     # Allow null for push sources
     harvester = models.ForeignKey('Harvester', null=True, on_delete=models.CASCADE)
     harvester_kwargs = JSONField(null=True, blank=True)
+    harvest_interval = models.DurationField(default='1 day')
+    harvest_after = models.TimeField(default='02:00')
+    backharvesting = models.BooleanField(default=False)
```

Comment:
Is this `backharvesting` as in we are currently backharvesting this source? Or is this `backharvested` as in we have backharvested this source. Or `backharvest` as in we have decided to backharvest this source.